definitions/npm/angular_v1.5.x/flow_v0.47.x-/test_angular_interval_log_v1.5.x.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import type { $Interval, $Log, AngularPromise } from 'angular';
+
+function testInterval($interval: $Interval) {
```

Comment:
Since you're creating a new test file, can you use `describe/it` for better test organization?

Reply:
Yes, good point.

Reply:
Actually, do you have any ideas how to get that to work with angular's dependency injection? I suspect that's why most the other files don't have describe/it...

Reply:
Never mind, figured it out. No need for injection, just need to name the args right.